archunit/src/main/java/com/tngtech/archunit/lang/syntax/elements/ClassesShould.java

```diff
@@ -821,6 +821,9 @@
      * {@link ArchRuleDefinition#noClasses() noClasses()}.{@link GivenClasses#should() should()}.{@link #accessClassesThat()}.{@link ClassesThat#haveFullyQualifiedName(String) haveFullyQualifiedName(String)}
      * </code></pre>
      *
+     * NOTE: 'access' refers only to violations by real accesses, i.e. accessing a field, and calling a method.
+     * Compare with {@link #dependOnClassesThat()} (DescribedPredicate)} that catches a wider variety of violations.
```

Comment:
Should the `(DescribedPredicate)}` be here?

Reply:
Oh true, good spot, I don't think so :joy:

Reply:
That looks like the typical autocorrect IntelliJ always gets me with :wink:

Reply:
I took the liberty to fix that up :+1: